drivers/can/Kconfig

```diff
@@ -20,6 +20,12 @@ module = CAN
 module-str = CAN
 source "subsys/logging/Kconfig.template.log_config"
 
+config CAN_SHELL
+	bool "Enable CAN Shell"
+	depends on SHELL
+	help
```

Comment:
About the device shell, what about enabling it here unconditionally like this
`select DEVICE_SHELL`

Reply:
Since the device-shell is not necessarily needed, I won't.

Reply:
Ok, that is fine, just pondering about it.